lib/themes/dosomething/paraneue_dosomething/js/campaign/OrganDonation.js

```diff
@@ -0,0 +1,84 @@
+const $ = require('jquery');
+import setting from '../utilities/Setting';
+import Takeover from 'takeover/Takeover';
+
+class OrganDonation extends Takeover {
+  constructor($takeoverContainer) {
+
+    super($takeoverContainer);
+
+    this.zipCodeButton = $('.submit-postal-code');
+    this.registrationButton = $('.submit-registration');
+    this.backButton = $('.takeover__screen .back-button');
+
+    this.set();
+  }
+
+  /*
+   * Handles the zip code request screen.
+   */
+  submitZipCode() {
+    this.zipCodeButton.off('click');
```

Comment:
@DFurnes cause I want to reset all the click handlers when a screen is active. I'm using CSS transitions so nothing ever leaves the DOM so, if you go back and forth between screens, I wanted to reset the state of the buttons.